paasta_tools/api/api_docs/oapi.yaml

```diff
@@ -1627,6 +1627,52 @@ paths:
       summary: Get mesos task of service_name.instance_name by task_id
       tags:
       - service
+  /services/{service}/{instance}/remote_run:
+    post:
+      operationId: remote_run
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                interactive:
+                  type: bool
+                user:
+                  type: string
+                image:
+                  type: string
+              required:
+               - interactive
```

Comment:
Still figuring out what parameters will be needed, so this doesn't match swagger right now